public/js/components/ObjectInspector.js

```diff
@@ -131,7 +131,7 @@ const ObjectInspector = React.createClass({
     let objectValue;
     if (nodeHasProperties(item) || nodeIsPrimitive(item)) {
       const object = item.contents.value;
-      objectValue = Rep({ object });
+      objectValue = Rep({ object, mode: "tiny" });
```

Comment:
@jasonLaster I don't know enough about the desired output across the tree and the way reps are being used here to have an opinion.